components/table/Table.tsx

```diff
@@ -28,13 +28,15 @@ export default function Table({
       <div>
         <TableTop title={title} description={description} />
         <div className="flex justify-between items-center border-b-2 px-4 py-2 w-full">
-          <SearchFilterSort
-            showDownloadButton={showDownloadButton}
-            apiContext={apiContext}
-            removeSearch={removeSearch}
-          />
+          <Suspense>
+            {' '}
```

Comment:
I guess we can remove the empty quots.

Reply:
same here wasn't added by me but added after running `git commit`

Reply:
looks like it is removed this time